drivers/hdc1000/Kconfig

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_HDC1000
+    bool "Configure HDC1000 driver"
+    depends on MODULE_HDC1000
+    help
+        Configure the HDC1000 driver using Kconfig.
+
+if KCONFIG_MODULE_HDC1000
+
+config HDC1000_I2C_ADDRESS
+    hex "HDC1000 I2C ADDRESS"
+    range 0x40 0x43
+    default 0x43
+    help
+        Change the I2C Slave Address.HDC1000 allows for upto 4 devices on Single Bus.
+        The value depends on the state of ADR1 and ADR0 Pins.
+        For more information refer section 8.5.1 in Datasheet (SNAS643C).
+
+config HDC1000_CONVERSION_TIME
+    int "HDC1000 CONVERSION TIME"
+    range 13000 26000
```

Comment:
Where does this range come from?

Reply:
The math for two sensors is 26000 , for using one this could be halved since the conversion time would be halved. I could reduce it even further ( not sure of doing it since I dont have the sensor ). You can find the reference below

https://e2e.ti.com/support/sensors/f/1023/t/403603?HDC1000-Response-Time

Reply:
I don't think we should place a range here, as there is no actual constraint. The information in the `help` section gives a hint on possible values, but it's up to the user to decide. Let's remove it.

Reply:
Okay. I shall remove the range and just have a default value.